source/sass/blocks/example.scss

```
@@ -33,9 +40,66 @@
 
 .example__slider {
   text-align: center;
+  display: flex;
 }
 
 .example__button {
   border-width: 0;
   padding: 0;
 }
+
+@media (min-width: $tablet-width) {
+  .example {
+    grid-template-columns: 185px 185px 320px;
+  }
+
+  .example__title {
+    grid-column: 1/-1;
+  }
+
+  .example__text {
+    grid-column: 1/-1;
+  }
+
+  .example__table {
+    grid-column: 1/3;
+  }
+
+  .example__price {
+    grid-column: 3/4;
+  }
+
+  .example__slider {
+    grid-column: 1/-1;
+    justify-content: center;
+  }
+}
+
+@media (min-width: $desktop-width) {
+  .example {
+    grid-template-columns: 560px 560px;
```

Лучше конечно не в абсолютных единицах.
```suggestion
    grid-template-columns: 1fr 1fr;
```